docs/src/rules/no-await-in-loop.md

```diff
@@ -74,18 +107,60 @@ async function foo(things) {
   const results = [];
   for (const thing of things) {
     // Bad: each loop iteration is delayed until the entire asynchronous operation completes
-    results.push(await bar(thing));
+    results.push(await doAsyncWork(thing));
   }
-  return baz(results);
+  return results;
 }
 ```
 
 :::
 
 ## When Not To Use It
 
-In many cases the iterations of a loop are not actually independent of each-other. For example, the
-output of one iteration might be used as the input to another. Or, loops may be used to retry
-asynchronous operations that were unsuccessful. Or, loops may be used to prevent your code from sending
-an excessive amount of requests in parallel. In such cases it makes sense to use `await` within a
+In many cases the iterations of a loop are not actually independent of each other, and awaiting in
+the loop is correct. As a few examples:
+
+* The output of one iteration might be used as the input to another.
+
+    ```ts
```

Comment:
I think you mean `js` here, or was it intentional?

Reply:
muscle memory 😆